src/utils/dateutils.py

```diff
@@ -6,5 +6,5 @@ def get_day_month(date: datetime) -> tuple:
 
 
 def get_current_day_month(utc_hour_offset: str) -> tuple:
-    current_date = datetime.utcnow() + timedelta(int(utc_hour_offset))
```

Comment:
With the following code: 
`timedelta(int(utc_hour_offset))`
We are adding days, for that reason, it was decided to specify the parameter we need to change, in this case, hours.